### PR TITLE
Create account w/ recovery key (email, phone, seed phrase)

### PR DESCRIPTION
**Pull request by Patrick1904**
Now creates account using the **recovery key** (seed phrase, phone, email). Once account creation is successful, it adds a new key to the account and localStorage. The result is one recovery key, and one localStorage key.

Dependency: https://github.com/near/near-contract-helper/pull/214 (https://near-contract-helper-2fa.onrender.com)

### CODE REVIEW

File: src/utils/wallet.js [hunk 1]
```diff
@@ -577,16 +582,22 @@ class Wallet {
...
 
     async initializeRecoveryMethod(accountId, method, isNew) {
+        const { seedPhrase } = generateSeedPhrase()
+
         if (isNew) {
-            return await sendJson('POST', ACCOUNT_HELPER_URL + '/account/initializeRecoveryMethodForTempAccount', {
+            await sendJson('POST', ACCOUNT_HELPER_URL + '/account/initializeRecoveryMethodForTempAccount', {
                 accountId,
                 method,
+                seedPhrase: [seedPhrase]
```
Comment (mattlockyer) [inline src/utils/wallet.js:591]: Can you not pass in array? Just use:
```
...
seedPhrase,
...
```

Comment (Patrick1904) [reply src/utils/wallet.js:591]: Screen Shot 2020-08-24 at 10 44 18 PM

@mattlockyer did I misunderstand you?

**Comment by Patrick1904** [comment]
> I'm good with this. Can someone remind me where we landed on save the key in localStorage in case there's a connection issue with RPC?
> 
> I don't think we have it here, but it's not blocking since the user has recovery link. Could be an improvement though.

Discussed w/ @kcole16 yesterday and we decided to skip for this PR.

**Comment by vgrichina** [comment]
@mattlockyer @kcole16 How did you test this given that Render wasn't pointed to updated contract helper?

Note that I've updated this PR's config on Render to actually point to contract helper.

https://wallet.testnet.near.org/recover-with-link/test-841.testnet/avocado%20educate%20angry%20hurt%20trial%20hotel%20birth%20winner%20civil%20catalog%20casino%20clever

**Comment by kcole16** [comment]
@vgrichina To test, I checked out the branch and updated the contract helper URL locally to point to Matt’s PR.

**Comment by mattlockyer** [comment]
You can add a .env in wallet root with
```
REACT_APP_ACCOUNT_HELPER_URL="https://near-contract-helper-2fa.onrender.com"
```

Apologies for not being clear. Should we add this to readme? Not really intended for people outside our org @kcole16

**Pull request merged**